src/main/scala/sigmastate/Values.scala

```diff
@@ -647,14 +651,14 @@ object Values {
     lazy val tpe: SFunc = SFunc(args.map(_._2), body.tpe)
     val opCode: OpCode = FuncValueCode
     /** This is not used as operation, but rather to form a program structure */
-    def opType: SFunc = Value.notSupportedError(this, "opType")
+    override def opType: SFunc = SFunc(Vector(), tpe)
```

Comment:
@greenhat this method is not supposed to be used. It is not defined for this node.

Reply:
I've used it in `RuntimeCosting` for `FuncValue` [here, in `costOf()`](http://github.com/ScorexFoundation/sigmastate-interpreter/blob/ad98d80a40e119097a4da0f7e0bab6d63e93e7e3/src/main/scala/sigmastate/eval/RuntimeCosting.scala#L1296) . I did not find a better way than to "enable" it.